Makefile

```diff
@@ -57,6 +57,9 @@ hooks:
 	echo "#!/bin/sh" >.git/hooks/pre-commit
 	echo "raco fmt -i \$$(find . -name '*.rkt')" >>.git/hooks/pre-commit
 
+format:
```

Comment:
C'mon dude call it `fmt`, literally more confusing this way :)

Reply:
Oof right